common/randombytes.h

```diff
@@ -1,13 +1,26 @@
-#ifndef RANDOMBYTES_H
-#define RANDOMBYTES_H
-#include <stdint.h>
+#ifndef sss_RANDOMBYTES_H
+#define sss_RANDOMBYTES_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
 
 #ifdef _WIN32
+/* Load size_t on windows */
 #include <crtdefs.h>
 #else
 #include <unistd.h>
-#endif
+#endif /* _WIN32 */
 
-int randombytes(uint8_t *buf, size_t n);
 
+/*
+ * Write `n` bytes of high quality random bytes to `buf`
+ */
+#define randombytes     PQCLEAN_randombytes
+int randombytes(void *buf, size_t n);
```

Comment:
This should be a `uint8_t *`?

Reply:
Yeah I guess this is an API change otherwise.